src/outlineIcons.ts

```diff
@@ -1,49 +1,102 @@
-import * as Icons from "@heroicons/react/outline"
+import * as Icons from "@heroicons/react/24/outline"
```

Comment:
hmm, I'm thinking this might be another option add, meaning, it should be asked on the lookup function. What do you think?

Reply:
I think the only options they have are 24/outline, 24/solid and "mini" which is 20/solid. So maybe we could just add a third option to `IconFormat`. What do you think?

```
export type IconFormat = "solid" | "outline" | "mini"
```

Reply:
Oh, if that's the case, then yes, sounds good.

Reply:
Nice. I've added that as well then. 👍